kie-drools-wb/kie-drools-wb-distribution-wars-cdi1.0/pom.xml

```diff
@@ -343,6 +343,11 @@
       <groupId>org.jbpm</groupId>
       <artifactId>jbpm-services-ejb-timer</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.codehaus.woodstox</groupId>
+      <artifactId>stax2-api</artifactId>
+      <version>3.1.1</version>
+    </dependency>
```

Comment:
I thought we are enforcing that the artifact version needs to be in `dependencyManagement`. Looking at the POM, the enforcer rule checking this was disable which is I think bad. We need to enable it again and move this version into `dependencyManagement` section (or directly into the kie-p-w-d).